AFQ/tests/test_api.py

```diff
@@ -180,6 +180,23 @@ def test_AFQ_custom_tract():
 
 
 @pytest.mark.nightly2
+def test_AFQ_no_derivs():
```

Comment:
Should this test really be marked as nightly? It probably runs pretty quickly (?)

Reply:
test_AFQ_init is also marked nightly. Initializing an afq object takes about a minute I think, and if we have a lot of these tests, they can add up. To reduce time to make afq object, see: https://github.com/yeatmanlab/pyAFQ/issues/448

Reply:
OK - that makes sense. Merging!